tools/clang/unittests/HLSL/VerifierTest.cpp

```diff
@@ -243,6 +244,10 @@ TEST_F(VerifierTest, RunEnums) {
   CheckVerifiesHLSL(L"enums.hlsl");
 }
 
+TEST_F(VerifierTest, RunFunctionTemplateDefault) {
+  CheckVerifiesHLSL(L"function-template-default.hlsl");
```

Comment:
where is this hlsl file located?

Reply:
Doh!

I didn't add it to the commit. Fixed in d46e73e3d60b33c9548b1f98ca77ded985f8ee38